tutorials/01_Basic_QA_Pipeline.ipynb

```diff
@@ -4,62 +4,38 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Build Your First QA System\n",
+    "# Build Your First Question Answering System\n",
     "\n",
-    "<img style=\"float: right;\" src=\"https://upload.wikimedia.org/wikipedia/en/d/d8/Game_of_Thrones_title_card.jpg\">\n",
+    "- **Level**: Beginner\n",
+    "- **Time to complete**: 20 minutes\n",
```

Comment:
about these times - have you actually tested them? It's important that we give real times here.